scripts/install/mylar.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+# Brett 2021
+#shellcheck source=sources/functions/pyenv
+. /etc/swizzin/sources/functions/pyenv
+#shellcheck source=sources/functions/utils
+. /etc/swizzin/sources/functions/utils
+app_name="mylar"
+if [ -z "$MYLAR_OWNER" ]; then
+    if ! MYLAR_OWNER="$(swizdb get "$app_name/owner")"; then
+        MYLAR_OWNER="$(_get_master_username)"
+        echo_info "Setting ${app_name^} owner = $MYLAR_OWNER"
+        swizdb set "$app_name/owner" "$MYLAR_OWNER"
+    fi
+else
+    echo_info "Setting ${app_name^} owner = $MYLAR_OWNER"
+    swizdb set "$app_name/owner" "$MYLAR_OWNER"
+fi
+user="$MYLAR_OWNER"
+swiz_configdir="/home/$user/.config"
+app_configdir="$swiz_configdir/${app_name^}"
+app_configfile="$app_configdir/config.ini"
+app_group="$user"
+app_port="$(port 7000 11000)"
+app_servicefile="$app_name.service"
+app_dir="/opt/${app_name^}"
+app_binary="${app_name^}"
+#Remove any dashes in appname per FS
+app_lockname="${app_name//-/}"
+echo_info "Setting ${app_name^} port = ${app_port}"
+swizdb set "mylar/port" "${app_port}"
+
+if [ ! -d "$swiz_configdir" ]; then
+    mkdir -p "$swiz_configdir"
+fi
+function install_mylar() {
+    chown "$user":"$app_group" "$swiz_configdir"
+    # deps
+    apt_install python3 git
```

Comment:
@liaralabs These should be handled in the python functions already and system-wide dependencies, no?